internal/boxcli/midcobra/midcobra.go

```diff
@@ -81,6 +81,7 @@ func (ex *midcobraExecutable) Execute(ctx context.Context, args []string) int {
 		var exitErr *exec.ExitError
 		var userExecErr *usererr.ExitError
 		if errors.As(err, &userExecErr) {
+			ux.Ferror(ex.cmd.ErrOrStderr(), userExecErr.Error()+"\n")
```

Comment:
Suggestion. Also print the Stderr since it may have other info.

We do similar in midcobra/debug.go: https://github.com/jetpack-io/devbox/blob/42d2f713fde840558faeb9d404b8f5e9e2f697dd/internal/boxcli/midcobra/debug.go#L67